src/app.css

```
@@ -121,6 +121,52 @@
+      &[data-image-state='loading'] {
+        @apply min-h-32 bg-gray-300 dark:bg-gray-600;
+
+        img {
+          @apply opacity-0;
+        }
+      }
+
+      &[data-image-state='loaded'] {
+        img {
+          @apply opacity-100 transition-opacity ease-out;
+          transition-duration: var(--oa-transition-duration);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)